index.js

```diff
@@ -89,4 +97,25 @@ module.exports = function (processors, options) {
   }
 
   return stream
+})
+
+
+function withConfigLoader(cb) {
+  return function (plugins, options) {
+    if (typeof plugins === 'undefined') {
+      return cb(postcssLoadConfig)
```

Comment:
@michael-ciniawsky I checked the webpack loader, and it injects `webpack` into the ctx, so I decided to inject `file` there that is a gulp file. I don't like an idea of merging the context options and file options into the same `ctx` namespace, because they may override each other. I also added a callback that allows to configure `gulp-postcss` directly from the gulp file.